src/js/tech/html5.js

```diff
@@ -469,7 +469,26 @@ class Html5 extends Tech {
    * @return {Number}
    * @method duration
    */
-  duration() { return this.el_.duration || 0; }
+  duration() {
+    // Android Chrome will report duration as Infinity for VOD HLS until after
+    // playback has started, which triggers the live display erroneously.
+    // Return NaN if playback has not started and trigger a durationupdate once
+    // the duration can be reliably known.
+    if (this.el_.duration === Infinity &&
+      browser.IS_ANDROID && browser.IS_CHROME) {
+      if (this.el_.currentTime === 0) {
+        const checkProgress = () => {
+          if (this.el_.currentTime > 0) {
+            this.trigger('durationchange');
```

Comment:
Oh, I see what you mean.
I believe we should get at least one `durationchange` from the video loading. When we get a `durationchange` from the tech, [the player does a `techGet` on duration](https://github.com/videojs/video.js/blob/fc72fb90e7661594c0ea090e0e1e9838f8ec9486/src/js/player.js#L1013-L1015) so this gets called.
Though, it would be interesting to know whether it's possible for `durationchange` not to trigger and also when it triggers.

(not as useful but still leaving here: Our duration display updates itself and [calls `player.duration()`](https://github.com/videojs/video.js/blob/fc72fb90e7661594c0ea090e0e1e9838f8ec9486/src/js/control-bar/time-controls/duration-display.js#L54) on [`durationchange`](https://github.com/videojs/video.js/blob/fc72fb90e7661594c0ea090e0e1e9838f8ec9486/src/js/control-bar/time-controls/duration-display.js#L21).)